composer.json

```diff
@@ -63,6 +63,8 @@
             "./third_party/bin/phpcs --standard=tests/rulesets/phpcs_ruleset.xml application/models/ArchivedTableSettings.php",
             "./third_party/bin/phpcs --standard=tests/rulesets/phpcs_ruleset.xml application/libraries/ExtensionInstaller",
             "./third_party/bin/phpcs --standard=tests/rulesets/phpcs_ruleset.xml application/helpers/update/updatedb_helper.php",
+            "./third_party/bin/phpcs --standard=tests/rulesets/phpcs_ruleset.xml application/core/plugins/ExpressionAnswerOptions/ExpressionAnswerOptions.php",
+            "./third_party/bin/phpcs --standard=tests/rulesets/phpcs_ruleset.xml application/core/plugins/ExpressionAnswerOptions/AnswerOptionsFunctions.php",
```

Comment:
Can you try phpmd_ruleset_strict.xml, please? It sets a limit on function length of 60 LOC.

Reply:
You know you can update file directly ?

Reply:
Hm, gonna try